service/frontend/workflowHandler_test.go

```diff
@@ -254,6 +254,24 @@ func (s *workflowHandlerSuite) TestDisableListVisibilityByFilter() {
 	assert.Equal(s.T(), errNoPermission, err)
 }
 
+func (s *workflowHandlerSuite) TestPollForTask_Failed_ContextTimeoutTooShort() {
+	config := s.newConfig()
+	wh := s.getWorkflowHandler(config)
+	wh.metricsClient = wh.Service.GetMetricsClient()
+	wh.startWG.Done()
+
+	shortCtx, cancel := context.WithTimeout(context.Background(), minLongPollTimeout-time.Millisecond)
+	defer cancel()
+
+	_, err := wh.PollForDecisionTask(shortCtx, &shared.PollForDecisionTaskRequest{})
+	assert.Error(s.T(), err)
```

Comment:
This should be written as `s.Error(err)`

Reply:
Yes, this is my original way of writing this. But it seems like other tests in this file are using assert, so I followed their pattern.